### PR TITLE
changes in forgot and reset password and some frontend changes

### PR DESCRIPTION
**Pull request by Parul76**

### CODE REVIEW

**Comment by Parul76** [comment]
I just push my changes because I was not able to pull new changes in my branch. but you can remove usercontroller.cs and userservice.cs from my pull request if you want.

File: mrscake/src/components/ResetPassword.jsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    state = 
+    {    
+      password: '',
+    }
+
+    
+    render() {
+         return <Form method="post">
+            <div className="app flex-row align-items-center">
+                <Container><br/><br/><br/>
```
Comment (Alina-dev-front) [inline mrscake/src/components/ResetPassword.jsx:15]: <br><br> = nononono

Comment (Alina-dev-front) [inline mrscake/src/components/ResetPassword.jsx:15]: "<br><br>" = nonono

Comment (Alina-dev-front) [inline mrscake/src/components/ResetPassword.jsx:15]: this was supposed to be padding?

File: mrscake/src/components/ResetPassword.jsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+class ResetPassword extends Component {
+    state = 
+    {    
+      password: '',
+    }
+
+    
+    render() {
+         return <Form method="post">
```
Comment (Alina-dev-front) [inline mrscake/src/components/ResetPassword.jsx:13]: what is method "post" here?

**Review by Alina-dev-front — APPROVED** [review]

**Pull request merged**